task_4/add-methods.js

```diff
@@ -0,0 +1,9 @@
+const Dog=function (name, color, numLegs){
+    this.name=name;
+    this.color=color;
+    this.numLegs=numLegs;
+    this.describe=function(){
+        return 'the dogs name is '+this.name+', his color is'+this.color+' and he has' +this.numLegs+ 'legs';
+    }
+}
+export default Dog
```

Comment:
Kada se dodaju metode losa je praksa cuvati metodu u samom objektu, mnogo je bolje dodati metodu u prototype od funkcije. Vise o tome mozes da procitas ovde. 
https://medium.com/better-programming/prototypes-in-javascript-5bba2990e04b
```suggestion
Dog.prototype.describe = function(){
        return 'the dogs name is '+this.name+', his color is'+this.color+' and he has' +this.numLegs+ 'legs';
    }
```